### DISCUSSION TITLE
Patron Saint

### DISCUSSION THREAD

**Discussion by rl1234567890** [DivinumOfficium/divinum-officium#2669] [Ideas]
On November 13, it was the feast of St. Didacus, who is the patron saint of America's Finest City. In the copy of Divinum Officium I run on my computer, I modified the rank of web/www/horas/Latin/Sancti/11-13.txt  from
S. Didaci Confessoris;;Semiduplex;;2;;vide C5
to 
S. Didaci Confessoris;;Duplex I classis;;6;;ex C5
Maybe a more general and elegant way do this for their local patron saints would be useful to the users.

**Comment by fiapps** [comment]
The ideal would be to support local calendars and calendars of religious institutes, and let the user select one or both, but this probably has to wait until the code base has been considerably cleaned up.
